src/os/bluestore/bluestore_tool.cc

```diff
@@ -316,6 +316,13 @@ int main(int argc, char **argv)
       cerr << "readdir in root failed: " << cpp_strerror(r) << std::endl;
       exit(EXIT_FAILURE);
     }
+
+    r = ::mkdir(out_dir.c_str(), 0755);
```

Comment:
We should make sure the dir does not exist before creating? Otherwise this is going to fail..

Reply:
thank for your review, I fixed another mkdir too.